### PR TITLE
lathes now print empty welders and extinguishers

### PR DESCRIPTION
**Pull request by Fikou**
## About The Pull Request
welders (other than experimental) printed by lathes now start with 0 fuel and have to be refueled with a welding tank
same with fire extinguishers

## Why It's Good For The Game
iron and glass dont create welding fuel or water, and it makes the tanks a bit more valuable

## Changelog
:cl:
add: lathes now print empty welders and extinguishers
/:cl:

### CODE REVIEW

**Comment by Sealed101** [comment]
shiptested dot com

**Comment by Sealed101** [comment]
on a more serious note: this could go either way, really. on shiptest (though i've played on voidcrew (RIP) but i imagine it's not too different on this subject matter) it's a necessity and makes for some fun gameplay quirks and workarounds. can't quite say it would have the same effect here unless it makes cargo not sell every goddamn tank in existence and improves the chance of someone actually sabotaging these items with plasma or chemicals (are extinguishers event susceptible to shenanigans? can't say)

**Comment by timothymtorres** [comment]
+1 for the consistency aspect

**Comment by Overuseddart** [comment]
Do extinguishers have different effects for the material like toolboxes?

**Comment by moocowswag** [comment]
do department lathes still require beakers to build?

maybe could have them optionally take water/welding fuel to prefuel printed items that use them?

**Comment by RedBaronFlyer** [comment]
Honestly, I'm okay with this change if we could ship in fuel and water tanks.

From bounties to cargo techs selling shit to assistants blowing them up in maints, the station will usually be out of welding and water tanks (welding fuel tanks especially) within forty minutes most of the time.

**Comment by thgvr** [comment]
I have no idea how tg has gone this long without this change, honestly. I can't remember the last time I saw someone refuel their welder at a fuel tank.

**Comment by Overuseddart** [comment]
Blob buff, mass printing welders is the crews first response to a blob

**Comment by Overuseddart** [comment]
> Honestly, I'm okay with this change if we could ship in fuel and water tanks via cargo.
> 
> The station will usually be out of welding and water tanks (welding fuel tanks especially) within forty minutes most of the time due to bounties, cargo techs selling everything not nailed down, and to bored assistants intentionally blowing them up because they're bored.

You already can

**Comment by RedBaronFlyer** [comment]
> > Honestly, I'm okay with this change if we could ship in fuel and water tanks via cargo.
> > The station will usually be out of welding and water tanks (welding fuel tanks especially) within forty minutes most of the time due to bounties, cargo techs selling everything not nailed down, and to bored assistants intentionally blowing them up because they're bored.
> 
> You already can

Oh, huh, I never knew that, despite playing CT for like 250 hours. My bad!

File: code/game/objects/items/extinguisher.dm [hunk 1]
```diff
@@ -18,7 +18,10 @@
 	attack_verb_simple = list("slam", "whack", "bash", "thunk", "batter", "bludgeon", "thrash")
 	dog_fashion = /datum/dog_fashion/back
 	resistance_flags = FIRE_PROOF
+	/// The max amount of water this extinguisher can hold.
 	var/max_water = 50
+	/// Does the welder extinguisher start with water.
+	var/starting_water = TRUE
 	var/last_use = 1
```
Comment (ArcaneMusic) [inline code/game/objects/items/extinguisher.dm:25]: Care to autodoc the rest of these since you're in the neighborhood

**Comment by moocowswag** [comment]
I was just informed that grinding cable coil gives you copper and...you guessed it, cable coil is printed out of iron and glass from lathes. This pr has so many implications lol.

**Comment by Fikou** [comment]
> I was just informed that grinding cable coil gives you copper and...you guessed it, cable coil is printed out of iron and glass from lathes. This pr has so many implications and need for follow ups lol

no it doesnt lol

**Comment by Overuseddart** [comment]
> I was just informed that grinding cable coil gives you copper and...you guessed it, cable coil is printed out of iron and glass from lathes. This pr has so many implications and need for follow ups lol

But we don’t have copper? If you wanted to go a step farther the station shouldn’t be made out of IRON walls, we should be in titanium Kevlar and steel, with the escape pods being aluminum. Then we could go more realistic, virology and medical need to be revamped. Then we can remove plasma for being unrealistic. Then mining. Then the entire game ( Nanotrasean and the Syndicate dont exist)

**Comment by moocowswag** [comment]
> But we don’t have copper?

 Copper is a chem but it doesnt matter cause i was apparently lied to

**Comment by Fikou** [comment]
> > But we don’t have copper?
> 
> Copper is a chem which is why it could have been relevant to the scope of this pr and whats it trying to achieve but it doesnt matter cause i was apparently lied to

you werent lied to. its just that the scope of this pr is to make it so you dont generate chemicals in things you print that use chemicals. cables dont use copper, theyre made of copper

**Pull request merged**